### DISCUSSION TITLE
Load StyleManager component after init in custom layout AppendTo no works

### DISCUSSION THREAD

**Discussion by jcenlo** [GrapesJS/grapesjs#4620] [Q&A] [answered]
I am trying to load managers outside of Grapesjs framework

i created a custom layout in react where i want to load components after the init of the editor

Once the editor is loaded my intention is generate the components(styles, classes, blocks...) when the layout is ready

i m not defining the styleManager configuration in the initial setting and i am setting the configuration after the layout call the partial

this is my code , i am rendering this component in the parent layout:

I am not sure if i am doing the integration well, but in the BlockManager works perfectly i think the **appendTo** in this case no works

<img width="717" alt="Screenshot 2022-09-27 at 17 49 04" src="https://user-images.githubusercontent.com/1485745/192574044-53c7b315-eec0-43ec-bf47-16031f4921f1.png">

Any help will be welcome

**Comment by artf** [comment, marked as answer]
It doesn't work because the Style Manager is activated immediately by default configuration here: https://github.com/artf/grapesjs/blob/c9465882ebb3ec91f33261d32019f2fdc6193d9c/src/panels/config/config.js#L59-L61
So, you can move `appendTo` of the StyleManager in the initial configuration object (probably the easiest solution) or you have to disable the command and activate it later
